documentation/modules/post/networking/gather/enum_brocade.md

```diff
@@ -36,7 +36,7 @@
 #### SSH Session with password-display off
 
 ```
-resource (brocade.rb)> use post/brocade/gather/enum_brocade
+resource (brocade.rb)> use post/networking/gather/enum_brocade
```

Comment:
Is this resource prompt needed? Or should this not be msf's default prompt followed by the module specific prompt? I don't see any references to loading up a resource file here so I'm confused how and why the documentation shows the prompt as `resource (brocade.rb)` as you don't seem to be loading a resource. This applies to all prompts in this documentation.

Reply:
See comment on resource files in the main chat. This was a mistake on my part.